include/shellstrings.h

```diff
@@ -0,0 +1,128 @@
+
+/* 
+ * String output functions to be used in the shell
+*/
+
+#ifndef INCLUDE_SHELLSTRINGS_H_
+#define INCLUDE_SHELLSTRINGS_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*
+ * shell_prompt - Default text input prompt string for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_prompt();
+
+/*
+ * shell_intro - Introduction text for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_intro();
+
+/*
+ * shell_save - Modification saving text for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_save();
+
+/*
+ * shell_print - Print edited text for viewing
+ *
+ * Parameters:
+ * 	- None.
```

Comment:
None? This conflicts with the declaration.

Reply:
specified